deploy/deploy.yaml

```diff
@@ -0,0 +1,103 @@
+---
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: multi-networkpolicy-tc
+rules:
+  - apiGroups: ["k8s.cni.cncf.io"]
+    resources:
+      - '*'
+    verbs:
+      - '*'
+  - apiGroups:
+      - ""
+    resources:
+      - pods
+      - namespaces
+    verbs:
+      - list
+      - watch
+      - get
+  - apiGroups:
+      - ""
+      - events.k8s.io
+    resources:
+      - events
+    verbs:
+      - create
+      - patch
+      - update
+---
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: multi-networkpolicy-tc
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: multi-networkpolicy
+subjects:
+- kind: ServiceAccount
+  name: multi-networkpolicy
+  namespace: kube-system
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: multi-networkpolicy-tc
+  namespace: kube-system
+---
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: multi-networkpolicy-tc-ds-amd64
+  namespace: kube-system
+  labels:
+    tier: node
+    app: multi-networkpolicy-tc
+    name: multi-networkpolicy-tc
+spec:
+  selector:
+    matchLabels:
+      name: multi-networkpolicy-tc
+  updateStrategy:
+    type: RollingUpdate
+  template:
+    metadata:
+      labels:
+        tier: node
+        app: multi-networkpolicy-tc
+        name: multi-networkpolicy-tc
+    spec:
+      hostNetwork: true
+      nodeSelector:
+        kubernetes.io/arch: amd64
+      tolerations:
+      - operator: Exists
+        effect: NoSchedule
+      serviceAccountName: multi-networkpolicy-tc
+      containers:
+      - name: multi-networkpolicy-tc
+        image: nvidia.com/multi-networkpolicy-tc:latest
+        imagePullPolicy: IfNotPresent
+        command: ["/usr/bin/multi-networkpolicy-tc"]
+        args:
+        - "--pod-rules-path=/var/lib/multi-networkpolicy-tc"
+        resources:
+          requests:
+            cpu: "100m"
+            memory: "150Mi"
+          limits:
+            cpu: "200m"
+            memory: "300Mi"
+        securityContext:
+          privileged: true
+          capabilities:
+            add: ["SYS_ADMIN", "SYS_NET_ADMIN"]
```

Comment:
why you need capabilities if it privlieged ?

Reply:
also i think we would want to avoid privileged.

will just add NET_ADMIN cap (and remove privileged) as it seems to be enough.